### PR TITLE
Add CAPI e2e aws

### PR DESCRIPTION
**Pull request by alexander-demicev**

### CODE REVIEW

File: ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-main.yaml [hunk 1]
```diff
@@ -66,6 +66,10 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-serial
+- as: e2e-aws-capi
```
Comment (stbenjam) [inline ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-main.yaml:69]: Tech preview jobs usually include a -techpreview suffix, so TRT knows these are experimental

Comment (alexander-demichev) [reply ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-main.yaml:69]: should I change the workflow name too or step name is enough?

Comment (stbenjam) [reply ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-main.yaml:69]: Whatever you prefer, the name of the job is all that matters to TRT

Comment (alexander-demichev) [reply ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-main.yaml]: thanks, I fixed the name

**Review by elmiko — COMMENTED** [review]
/lgtm

File: ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-main.yaml (hunk 1 above)
Comment (stbenjam) [reply ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-main.yaml]: Thank you!

**Pull request merged**